<launch>
    <!-- 启动激光雷达扫描点角度滤波器节点 -->
    <node pkg="amr_laser_filters" type="scan_filter" name="laser_filter" output="screen">
        <!-- 加载YAML配置文件 -->
        <rosparam file="$(find amr_laser_filters)/config/laser_filter.yaml" />
        <!-- 将节点的"scan"话题重映射到全局的"/scan"话题 -->
        <remap from="/scan" to="/scan"/> <!-- 原始激光雷达数据话题 -->
        <!-- 将节点的"filtered_scan"话题重映射到全局的"/filtered_scan"话题 -->
        <remap from="/filtered_scan" to="/filtered_scan"/> <!-- 过滤后的激光雷达数据 -->
    </node>
</launch>